src/Features/CSharp/Portable/CodeRefactorings/CSharpRefactoringHelpersService.cs

```diff
@@ -2,12 +2,13 @@
 
 using System.Composition;
 using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Host.Mef;
 
 namespace Microsoft.CodeAnalysis.CSharp.CodeRefactorings
 {
     [ExportLanguageService(typeof(IRefactoringHelpersService), LanguageNames.CSharp), Shared]
-    internal class CSharpRefactoringHelpersService : AbstractRefactoringHelpersService
+    internal class CSharpRefactoringHelpersService : AbstractRefactoringHelpersService<PropertyDeclarationSyntax, ParameterSyntax, MethodDeclarationSyntax, LocalDeclarationStatementSyntax, LocalFunctionStatementSyntax>
```

Comment:
nit: consider wrapping.  we generally will write this as:

```c#
 internal class CSharpRefactoringHelpersService : AbstractRefactoringHelpersService<
    PropertyDeclarationSyntax,
    ParameterSyntax,
    MethodDeclarationSyntax,
    LocalDeclarationStatementSyntax,
    LocalFunctionStatementSyntax>
```

Reply:
Yep, with the last one it's a bit too long.  It'll get removed anyway, tho.